cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
```

Comment:
The 'specPattern' configuration doesn't need to include the 'cypress/integration' prefix as it is the default. If you have your specs in a different directory, specify that path instead. Otherwise, you can omit it for conciseness.